app/views/layouts/state_file/question.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= yield :form_question %>
   <% end %>
 
-  <section class="slab question-layout">
```

Comment:
Adding a css class here that is dependent on the page allows us to do outer layout specific rules for each page